FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2BottomDrawerActivity.kt

```diff
@@ -412,7 +414,7 @@ private fun CreateDrawerWithButtonOnPrimarySurfaceToInvokeIt(
 ) {
     val scope = rememberCoroutineScope()
 
-    val drawerState = rememberBottomDrawerState(expandable = expandable, skipOpenState = skipOpenState)
+    val drawerState = rememberBottomDrawerState(initialValue = DrawerValue.Closed, expandable = expandable, skipOpenState = skipOpenState)
```

Comment:
we are not passing this initial value for drawer?

Reply:
Initializing Drawers with Open states is causing assertions to fail and leads to the UI hanging due to how the Demo Activities are initialized. Will fix the demo activity for drawer in the next PR.